my_compassion/templates/pages/my2_user_settings.xml

```diff
@@ -0,0 +1,624 @@
+                                                id="city"
+                                                name="city"
+                                                class="form-control"
+                                                t-att-value="partner.city"
+                                                t-att-data-original-value="partner.city"
+                                            />
+                                        </div>
+                                    </t>
+                                </div>
+                            </div>
+                            <div class="mb-3">
+                                <t t-call="theme_compassion_2025.FormFieldComponent">
+                                    <t t-set="label" t-value="'Address'" />
+                                    <t t-set="input_id" t-value="'street'" />
+                                    <div class="field-display" data-display-for="street">
+                                        <t t-esc="partner.street or ''" />
+                                    </div>
+                                    <div class="field-edit">
+                                        <input
+                                            type="text"
+                                            id="street"
+                                            name="street"
+                                            class="form-control"
+                                            t-att-value="partner.street"
+                                            t-att-data-original-value="partner.street"
+                                        />
+                                    </div>
+                                </t>
+                            </div>
+                            <div class="row mb-2">
+                                <div class="col-md-6">
+                                    <t t-call="theme_compassion_2025.FormFieldComponent">
+                                        <t t-set="label" t-value="'Phone number'" />
+                                        <t t-set="input_id" t-value="'phone'" />
+                                        <div class="field-display" data-display-for="phone">
+                                            <t t-esc="partner.phone or ''" />
+                                        </div>
+                                        <div class="field-edit">
+                                            <input
+                                                type="tel"
+                                                id="phone"
+                                                name="phone"
+                                                class="form-control"
+                                                t-att-value="partner.phone"
+                                                t-att-data-original-value="partner.phone"
+                                            />
+                                        </div>
```

Comment:
Partner can either have a phone, or a mobile phone, it wasn't present on the figma design, but both should be handled